src/app/components/shell/sidebar/AppShellSidebar.vue

```diff
@@ -9,6 +9,7 @@ import AppShellSidebarBackpack from './AppShellSidebarBackpack.vue';
 import AppShellSidebarContext from './AppShellSidebarContext.vue';
 import AppShellSidebarLibrary from './AppShellSidebarLibrary.vue';
 import AppShellSidebarMobile from './AppShellSidebarMobile.vue';
+import AppShellSidebarQuests from './AppShellSidebarQuests.vue';
```

Comment:
switch this sidebar to be async loaded so that it doesn't have to load the code until they switch.
actually, can you also do for library and backpack? I didn't realize we're not separating that code into different chunks